app/components/header/index.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+
+import { useRouter } from 'next/navigation'
+
+const Header = () => {
+  const router = useRouter()
+
+  const handleNewTodo = () => {
+    router.push('/new-item')
+  }
+  return (
+    <section className="py-4">
+      <button
+        className="border border-slate-300 text-slate-300 px-2 py-1 rounded hover:bg-slate-700 focus-within:bg-slate-700 outline-none"
+        onClick={handleNewTodo}
+      >
+        + New Todo
```

Comment:
Create a constants file and read this from the constant file

Reply:
Added the constants file and referring constants from it.